js/controllers/list/submissions/SubmissionsListPanel.vue

```diff
@@ -54,6 +73,14 @@ export default {
 			infoUrl: '',
 		};
 	},
+	computed: {
+		/**
+		 * Can the current user filter the list?
+		 */
+		currentUserCanFilter: function() {
+			return pkp.userHasRole(['manager', 'subeditor', 'assistant']);
```

Comment:
`js/load.js` in OJS/OMP. Eventually we'll probably build the `pkp` global from an export and can share code between OJS/OMP. But I'm keeping it simple for now.